src/prx/helpers.py

```diff
@@ -394,3 +406,19 @@ def timeit_wrapper(*args, **kwargs):
         return result
 
     return timeit_wrapper
+
+
+def compute_gps_leap_seconds(yyyy: int, doy: int):
```

Comment:
rename `compute_gps_utc_leap_seconds`?

Reply:
modified in #81, 9b52d2560987b9f449ec7a535e6a3fc461822526